### ISSUE TITLE
How to remove uninstalled packages?

### ISSUE THREAD

**Issue by GFoley83** [blittle/bower-installer#104]
Scenario:
1. I install a package with `bower install <some package> -S`
2. I then run `bower-installer` to install the main package file to a directory of my choosing via the install > path > js setting in `bower.json`
3. I no long want this file so I run `bower uninstall <some package> -S`
4. I run `bower-installer` again.

The package I removed is still present in the destination directory even though I uninstalled it. 

How do I get Bower-Installer to keep in sync with the dependencies listed in `bower.json`? 
In other words, how do I get Bower-Installer to delete packages that no longer exist/are required?

**Comment by RQuadling** [comment]
Run ```bower-installer --help``` to see the help.

```
---------------------------------------------------------------------
Bower Installer
---------------------------------------------------------------------
Tool for installing bower dependencies that won't include entire repos.
Although Bower works great as a light-weight tool to quickly install
browser dependencies, it currently does not provide much functionality
for installing specific "built" components for the client.
---------------------------------------------------------------------
Options:
--remove [r] - Remove the bower_components directory after execution.
--remove-install-path [p] - Remove the install destination paths before installing dependencies.
--help   [h] - Display this.
---------------------------------------------------------------------
```

So, run ```bower-installer -p``` to clean out the destination directory first.
